error

```javascript
1| let promise = new Promise((resolve, reject) => {
2|   //do something important
3|   setTimeout(() => {
4|     reject("error");
5|   }, 1000);
6| });
```
```javascript
1| // promise.then attach handler on success and error
2| promise
3|   .then(
4|     result => {console.log("Fulfilled: " + result)}, // result - argument resolve
5|     error => {console.log("Rejected: " + error)} // error - argument reject
6|   );
```